text/1522-conservative-impl-trait.md

```diff
@@ -471,7 +471,7 @@ trait Foo {
 
 Given the above code, there is no way for `bar` to choose a return type `X`
 that could fundamentally differ between instantiations of `Self`
-while still being instantiable with an arbitrary `U`.
```

Comment:
And I believe you. The magic words for me were "English derivational morphology"